blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md

```diff
@@ -0,0 +1,772 @@
+---
+title: HummingBoard Pulse with Yocto and Buildroot
+abstract: 'In this article, I will show you how to build and run the image from
+           Yocto Project and SolidRun at HummingBoard Pulse. Furthermore, you
+           can find here examples of which tools and configurations were used.'
+cover: /covers/yocto-vs-buildroot.png
+author: cezary.sobczak
+layout: post
+published: true
+date: 2020-04-08
+archives: "2020"
+
+tags:
+  - SolidRun
+  - Buildroot
+  - HummingBoard
+  - Yocto
+  - u-boot
+categories:
+  - Miscellaneous
+  - OS Dev
+
+---
+
+## Introduction
+In this article, I will describe differences between two types of system
+images, i.e. SolidRun and Yocto. I was using a HummingBoard Pulse as a platform
```

Comment:
SolidRun is a company (vendor) of the HB boards. It sounds like it is an image type?